File: TopGun/src/app/model/map-marker.ts
export class MapMarker {
    id: number;
    Longitud: number;
    Latitud: number;
    Name: string;
    Description: string;

    constructor(data: Partial<MapMarker>) {
        Object.assign(this, data);
    }

    public static empty(): MapMarker {
        return new MapMarker({
            id: 0,
            Longitud: 0,
            Latitud: 0,
            Name: '',
            Description: '',
        });
    }

}

File: MiracleMileAPI/MiracleMileAPI/Model/MapMarker.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MiracleMileAPI.Model
{
    public class MapMarker
    {
        public int Id { get; set; }
        public double Longitud { get; set; }
        public double Latitud { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }

    }
}
